src/pages/about.js

```diff
@@ -1,13 +1,18 @@
 import React, { useState } from 'react'
 import Head from 'next/head'
 import { PageContainer, MainContainer, ContentContainer } from '@/components/Layout'
+import { ActionSpan } from '@/components/Typography'
 import { Circle } from '@/drawings/circle'
 import { Example } from '@/drawings/example'
 
 export default function About () {
   const [isPaused, setIsPaused] = useState(true)
 
-  const togglePause = () => setIsPaused(paused => !paused)
+  const actionProps = {
+    onClick: () => setIsPaused(paused => !paused),
+    onMouseEnter: () => setIsPaused(false),
+    onMouseLeave: () => setIsPaused(true)
+  }
```

Comment:
In this case, every span should trigger the same actions. And the only action is _pause_.

A more complex canvas drawing might have multiple shapes we need to animate separately. I think this system can still support that. As long as each `ActionSpan` has a unique `id`, we can update the state of specific canvas things based on their IDs

```jsx
const [actions, setActions = useState({
  pointC: { isPaused: true },
  lineCE: { isPaused: true }
})

const actionProps = {
  onClick: (event) => setIsPaused(prevState => ({
    ...prevState,
    [event.target.id]: {
      paused: !prevState[event.target.id]
    }
  }))
}
```